crates/wash-cli/src/cmd/dev/mod.rs

```diff
@@ -362,10 +430,17 @@ pub async fn handle_command(
                     }
 
                     // Stop NATS
-                    if let Some(mut nats) = nats_child {
+                    if let Some(mut nats) = nats {
                         eprintln!("{} Stopping NATS...", emoji::HOURGLASS_DRAINING);
                         nats.kill().await?;
                     }
+
+                    // Stop nats-kv-secrets
+                    if let Some(mut nats_kv_secrets) = nats_kv_secrets {
+                        eprintln!("{} Stopping nats-kv-secrets...", emoji::HOURGLASS_DRAINING);
+                        nats_kv_secrets.kill().await?;
+                    }
```

Comment:
What are your thoughts on just embedding the secrets-nats-kv as a library here, so we don't have to spawn off yet another process?

Reply:
Ah if we do that, IMO we should do that as a part of `wash` -- I think it's been a long time coming that `wash` should have been able to spawn the secrets binary and enable secrets support from the beginning! IIRC someone noted reasons against it, but IMO let's handle the integration as a separate thing and build it into starting `wash`!

Will happily rebase this PR to that if we get it done before this is ready